### PR TITLE
chore(port) Port build.sbt to io.getspectrum organization, formatting changes

### PR DESCRIPTION
**Pull request by bcottam**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -20,37 +23,15 @@ val awsSnsArtifact          = "com.amazonaws"          %  "aws-java-sdk-sns"
 val mailArtifact            = "com.sun.mail"           %  "mailapi"                   % mailVersion
 val smtpArtifact            = "com.sun.mail"           %  "smtp"                      % mailVersion
 val slf4jApiArtifact        = "org.slf4j"              %  "slf4j-api"                 % slf4jVersion
-val slf4jSimpleArtifact     = "org.slf4j"              %  "slf4j-simple"              % slf4jVersion
-val scalatestArtifact       = "org.scalatest"          %% "scalatest"                 % "3.0.5"  % "test"
-val scalacheckArtifact      = "org.scalacheck"         %% "scalacheck"                % "1.13.5" % "test"
 val stubbornArtifact        = "com.krux"               %% "stubborn"                  % "1.3.0"
 
-scalaVersion in ThisBuild := scala212Version
+scalaVersion in ThisBuild := scala211Version
```
Comment (jnewman) [inline build.sbt:28]: 2.12 still giving us trouble?

File: core/src/main/scala/com/krux/hyperion/common/S3Uri.scala [hunk 2]
```diff
@@ -12,7 +12,7 @@ import com.krux.hyperion.datanode.S3DataNode
  * or using a builder such as s3 / "hyperion-bucket" / "some-path".
  */
 case class S3Uri(ref: String) {
-  require(ref.startsWith("s3://"), "S3Uri must start with s3 protocol.")
+  require(ref.startsWith("s3://") || ref.startsWith("s3n://"), "S3Uri must start with s3 protocol.")
```
Comment (jnewman) [inline core/src/main/scala/com/krux/hyperion/common/S3Uri.scala]: FYI https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-file-systems.html
> Previously, Amazon EMR used the S3 Native FileSystem with the URI scheme, s3n. While this still works, we recommend that you use the s3 URI scheme for the best performance, security, and reliability.

**Review by jnewman — APPROVED** [review]
LSTM, I don't think we should use `s3n` anymore.

File: build.sbt (hunk 1 above)
Comment (bcottam) [reply build.sbt:28]: yeah, I can mark it for cross-compile if you like.

Comment (jnewman) [reply build.sbt:28]: Nah

**Review by jnewman — APPROVED** [review]

**Pull request merged**